util/config.go

```diff
@@ -50,12 +51,20 @@ func AddGlobalConfigFiles(cfg *mybase.Config) {
 	// running the test happens to have a ~/.my.cnf, ~/.skeema, /etc/skeema, it
 	// it would affect the test logic.
 	if cfg.IsTest {
-		globalFilePaths = append(globalFilePaths, "fake-etc/skeema", "fake-home/.my.cnf")
+		if cfg.CLI.OptionValues["ignore-my-cnf"] == "1" {
```

Comment:
This is a bit unusual, since it would only work on the command-line. i.e. setting this option in a global config file effectively won't do anything (silently, even) because the global config files have not been processed yet by this point...

A good work-around may be to move the logic into the `for` loop later in this function... inside of the conditional `if strings.HasSuffix(path, ".my.cnf")`, you could use `cfg.GetBool` to check if the user does not want to parse .my.cnf, and `continue` if so. I believe that should also prevent the test coverage % reduction.

That work-around would at least allow this option to function if set in `/etc/skeema` or `/usr/local/etc/skeema`. However, it still would not work in `~/.skeema` since that file is parsed after `~/.my.cnf`, and ditto with individual `.skeema` files inside the schema repo. There's no straightforward solution for that, so it's fine as long as it's documented clearly in options.md.